src/main/java/frc/robot/commands/AutoPaths/GalacticSearch.java

```diff
@@ -21,6 +22,15 @@
   public static final double INTAKE_DELAY = 1.0;
   // delay for a second when we get to a choice point, to ensure conveyor can notice the ball
   private boolean galacticSearchDone = false;
+  ConveyorTalon conveyorTalon;
+  Intake intake;
+
+  @Override
+  public void end(boolean interrupted) {
+    super.end(interrupted);
```

Comment:
We need to call `super.end` to have the default actions happen, in addition to the custom code we want